config/run_adios2stream.py

```diff
@@ -0,0 +1,325 @@
+                   'square-disc-nurbs.mesh',
+                   'beam-hex-nurbs.mesh',
+                   'amr-quad.mesh -o 2',
+                   'amr-hex.mesh',
+                   ],
+
+         'ex14p': [# not supported 'star-mixed.mesh -o 2',
+                   # not supported 'fichera-mixed.mesh -s 1 -k 1',
+                   'star.mesh -o 2',
+                   'inline-quad.mesh -o 0',
+                   'escher.mesh -s 1',
+                   'fichera.mesh -s 1 -k 1',
+                   'square-disc-p2.vtk -o 2',
+                   'square-disc-p3.mesh -o 3',
+                   'square-disc-nurbs.mesh -o 1',
+                   'disc-nurbs.mesh -rs 4 -o 2 -s 1 -k 0',
+                   'pipe-nurbs.mesh -o 1',
+                   'inline-segment.mesh -rs 5',
+                   'amr-quad.mesh -rs 3',
+                   'amr-hex.mesh',
+                   ],
+         # Bugs as in ex6p
+         #   'ex15p': []
+
+         'ex16p': [# Not supported '-m ../data/fichera-mixed.mesh',
+                   # Seg fault '-m ../data/amr-hex.mesh -o 2 -rs 0 -rp 0',
+                   '-m ../data/inline-tri.mesh',
+                   '-m ../data/disc-nurbs.mesh -tf 2',
+                   '-s 1 -a 0.0 -k 1.0',
+                   '-s 2 -a 1.0 -k 0.0',
+                   '-s 3 -a 0.5 -k 0.5 -o 4',
+                   '-s 14 -dt 1.0e-4 -tf 4.0e-2 -vs 40',
+                   '-m ../data/fichera-q2.mesh',
+                   '-m ../data/escher-p2.mesh',
+                   '-m ../data/beam-tet.mesh -tf 10 -dt 0.1',
+                   '-m ../data/amr-quad.mesh -o 4 -rs 0 -rp 0',
+                   ],
+
+         'ex17p': ['beam-tri.mesh',
+                   'beam-quad.mesh',
+                   'beam-tet.mesh',
+                   'beam-hex.mesh',
+                   'beam-wedge.mesh',
+                   'beam-quad.mesh -rs 2 -rp 2 -o 3 -elast',
+                   'beam-quad.mesh -rs 2 -rp 3 -o 2 -a 1 -k 1',
+                   'beam-hex.mesh -rs 2 -rp 1 -o 2',
+                   ],
+
+         'ex18p': [# Seg fault '-p 1 -rs 1 -rp 1 -o 5 -s 6',
+                   '',
+                   '-p 1 -rs 2 -rp 1 -o 1 -s 3',
+                   '-p 1 -rs 1 -rp 1 -o 3 -s 4',
+                   '-p 2 -rs 1 -rp 1 -o 1 -s 3',
+                   '-p 2 -rs 1 -rp 1 -o 3 -s 3',
+                   ],
+
+         'ex19p': ['beam-quad.mesh',
+                   'beam-tri.mesh',
+                   'beam-hex.mesh',
+                   'beam-tet.mesh',
+                   'beam-wedge.mesh',
+                   ],
+         # Connectivity issues, can't refine data, works with full data
+         'ex20p': ['']
+         }
+
+
+def ArgParser():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("example", type=str, help='example executable')
+    args = parser.parse_args()
+    return args
+
+
+def Run(example):
+    if example.endswith('p'):
+        run_command = 'mpirun '
+        run_prefix = '-n 4 ' + example
```

Comment:
The './' is missing for example here for the parallel case.

Reply:
@rcarson3 Good catch, thanks!